code/src/index.css

```diff
@@ -5,9 +5,113 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  box-sizing: border-box;
+  background-color: white;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.happy-thoughts {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+ .thoughts {
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+} 
+
+.thought-cards {
+  border: 1px solid;
+  padding: 10px;
+  box-shadow: 10px 10px;
+  margin: 20px 20px;
+  padding: 20px 20px;
+  border: 2px solid black;
+  font-family: 'Cutive Mono', monospace;
+  overflow-wrap: break-word;
+}
+
+/*** THE MY-THOGHT-CARD ***/
+
+.new-thought {
+  background-color: whitesmoke; 
+}
+
+.form {
+  display: flex;
+  flex-direction: column; 
+}
+
+.inputfield {
+  border: 1px solid black;
+  height: 50px;
+}
+
+#countdown {
+  font-size: 0.8rem;
+}
+
+.send-button {
+  padding: 6px 10px;
+  border-radius: 16px;
+  background-color: lightpink;
+  margin-top: 16px;
+  width: 200px;
+}
+
+.send-button:focus {
+  outline: none;
+  }
+
+/** RECENT THOUGHTS ***/
+
+.likebutton {
+  background-color: lavender;
+  border-radius: 50%;
+  padding: 6px;
+  border: none;
+  text-decoration: none;
+} 
+
+.likebutton:focus {
+  outline: none;
+  background-color: lightpink; 
+}
+
+.thought-details {
+  display: flex;
+  justify-content: space-between;
+  color: grey;
+}
+
+.like-details {
+  display: flex;
+  justify-content: space-between;
+  width: 70px;
+  align-items: baseline;
+}
+
+/** MEDIA QUERY TABLET **/
+
+@media(min-width:668px){
+.thoughts {
+    width: 60%;
+} 
+}
+/** MEDIA QUERY DESKTOP **/
+
+@media(min-width:1024px){
+.thoughts {
+    width: 40%;
+} 
+}
+
```

Comment:
The media queries would look nice with some indention work. Now it's easy to read because there isn't so much in each query but if you added more lines it can be hard to read